packages/cli/src/commands/deploy/index.ts

```diff
@@ -357,6 +345,12 @@ export default async (client: Client): Promise<number> => {
     }
   }
 
+  // For repo-style linking, reset the path to the root of the repository
+  //console.log(link);
+  if (link.status === 'linked' && link.repoRoot) {
+    cwd = join(link.repoRoot);
```

Comment:
I don't think the `join()` is doing anything. Any truthy value for `repoRoot` will be just end up being returned by `join()`. Did you mean `resolve()`?

Reply:
You're right, good catch.

```suggestion
    cwd = link.repoRoot;
```